honeybee/boundarycondition.py

```diff
@@ -153,6 +175,8 @@ def __init__(self, sun_exposure=True, wind_exposure=True,
                 This input can also be the word 'autocalculate' to have the view
                 factor automatically calculated.  Default: 'autocalculate'.
         """
+        sun_exposure = True if sun_exposure is None else sun_exposure
```

Comment:
similar to what I said above. These checks can be removed.

Reply:
I just made a decision to have `sun_exposure` as a boolean in the schema.  This property is just much better described with a "yes or no" answer that the text values gets unnecessarily confusing, creating more room for errors than they eliminate.  However, I took your suggestion and used a `lower()` check for the `autocalculate` parameter of the `view_factor`.